network/harvester.py

```diff
@@ -155,12 +158,13 @@ def main():
     parser.add_argument('--nodes', help='(optional) nodes json file')
     parser.add_argument('--output', help='output file', required=True)
     parser.add_argument('--thread-count', help='number of threads', type=int, default=16)
+    parser.add_argument('--mosaic-id', help='mosaic id', default=MAINNET_XYM_MOSAIC_ID)
     args = parser.parse_args()
 
     resources = load_resources(args.resources)
     blocks_per_day = 60 if 'nem' == resources.friendly_name else 120
     downloader = HarvesterDownloader(resources, int(args.days * 24 * blocks_per_day), args.nodes)
-    downloader.download(args.thread_count, args.output)
+    downloader.download(args.thread_count, args.output, args.mosaic_id)
```

Comment:
for proper handling, you'll probably also need to specify alias mosaic id [E74B99BA41F4AFEE] too

Reply:
Is this needed for Harvesting?  I saw the alias mosaic getting used for transfers but didnt change that.  I guess you want to update the pulling of transfers also?

Reply:
ah, ok, you're right. the single one should be fine for just account lookup.